### PR TITLE
Extend functions due to change in Amun

### PR DESCRIPTION
**Pull request by pacospace**

### CODE REVIEW

File: thoth/lab/inspection_report.py [hunk 1]
```diff
@@ -23,234 +23,96 @@
...
+    )
+
+def create_df_report(df: pd.DataFrame) -> pd.DataFrame:
+    """Show unique values for each column in the dataframe."""
+    dataframe_report = {}
+    for c_name in df.columns.values:
+        try:
+            unique_values = df[c_name].unique()
+            dataframe_report[c_name] = [unique_values] 
+        except Exception as e:
```
Comment (CermakM) [inline thoth/lab/inspection_report.py]: What kind of exception is thrown here?

File: thoth/lab/inspection_report.py [hunk 2]
```diff
@@ -23,234 +23,96 @@
...
-                key_counts[key] = len(set([k[key] for k in results_features[feature]]))
-                if explanation:
-                    if key_counts[key] != 1:
-                        explanation_summary[feature][key] = set([k[key] for k in results_features[feature]])
-
-            features_summary[feature] = key_counts
-
-    if explanation:
-        return _visualize_differences_in_inspection_results(explanation_summary, inspection_report_dict)
+            if len(_INSPECTION_JSON_DF_KEYS_FEATURES_MAPPING[class_features[0]]) > 1:
```
Comment (CermakM) [inline thoth/lab/inspection_report.py]: This can be moved up one level to `elif`

File: thoth/lab/inspection_report.py [hunk 3]
```diff
@@ -23,234 +23,96 @@
...
-                for sub_feature in sub_features:
-
-                    if sub_feature not in results_per_feature_per_batch[feature].keys():
-                        results_per_feature_per_batch[feature][sub_feature] = []
-
-                    for k, v in inspection_report_dict[batch][feature][sub_feature].items():
-                        results_per_feature_per_batch[feature][sub_feature].append(
-                            inspection_report_dict[batch][feature][sub_feature][k]
-                        )
             else:
```
Comment (CermakM) [inline thoth/lab/inspection_report.py]: This can be moved up one level.

File: thoth/lab/dependency_monkey.py [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+    dm_info_dict = {}
+    inspection_batch_identifiers = []
+    number_dm_ids = len(dm_ids)
+    current_dm_counter = 1
+    inspection_batch_counter = 0
+
+    if limit_results:
+        _LOGGER.info(f"Limiting results to {max_batch_identifiers_ids} to test functions!!")
+
+    for n, ids in enumerate(dm_ids):
```
Comment (CermakM) [inline thoth/lab/dependency_monkey.py]: `n` is unused

File: thoth/lab/inspection_report.py (hunk 1 above)
Comment (pacospace) [reply thoth/lab/inspection_report.py]: I'll create exception file for thoth-lab

**Pull request merged**